### PR TITLE
Release 0.15.1

### PR DESCRIPTION
**Pull request by uint**

### CODE REVIEW

**Review by chipshort — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,26 @@
...
+**Merged pull requests:**
+
+- Add Deque [\#807](https://github.com/CosmWasm/cw-plus/pull/807) ([chipshort](https://github.com/chipshort))
+- Cw1155-base public queries and move tests [\#804](https://github.com/CosmWasm/cw-plus/pull/804) ([ismellike](https://github.com/ismellike))
+- Add clear and is\_empty methods to Map [\#803](https://github.com/CosmWasm/cw-plus/pull/803) ([manu0466](https://github.com/manu0466))
+- SnapshotItem total, public query methods, and safe math [\#802](https://github.com/CosmWasm/cw-plus/pull/802) ([ismellike](https://github.com/ismellike))
+
+## [v0.15.1](https://github.com/CosmWasm/cw-plus/tree/v0.15.1) (2022-09-27)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.15.0...v0.15.1)
```
[the remark this answers is not in the document]
Comment (uint) [reply CHANGELOG.md:20]: Line 18 is the title, line 20 is a link to all the commits between 0.15.0 and 0.15.1

[the remark this answers is not in the document]
Comment (uint) [reply CHANGELOG.md:20]: Oh! I see now. Huh.

[the remark this answers is not in the document]
Comment (chipshort) [reply CHANGELOG.md:20]: What I mean is, they are already in line 7 and 9

[the remark this answers is not in the document]
Comment (uint) [reply CHANGELOG.md]: Good catch. Long story short, I suspect I got bitten by this bug:
https://github.com/github-changelog-generator/github-changelog-generator/issues/992

I'm removing the changelog update commit, merging this, and then I'll update the changelog as suggested in the ticket.

**Pull request merged**